### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_Screenreader:NVDA/Narrator is announcing Keybendings results twice

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#7014]
--- GitHub Tags:
-A11yMAS; -A11yTCS; -A11ySev3; -DesktopApp; -Win32; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Aug2022; -WCAG1.3.1; -NVDA; -ScreenReader; -Narrator; 

--- Environment and OS details:
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

--- Repro Steps:

1. Turn on NVDA
2. Open Visual studio code editor.
3. Navigate to setting icon
4. Select "Keyboard shortcuts options"
5. Seach name in search bar
6. Verify that screen reader announcing the results of keyboard shortcuts multiple twice or not.

--- Actual Result:
NVDA/Narrator is announcing Keybendings results twice. 

--- Expected Result:

NVDA/Narrator should not announce Keybendings results twice. 

--- Attachment:

![A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_NVDA](https://user-images.githubusercontent.com/100189407/186893090-e37dcd4d-f2bb-4264-aa62-4d970ef667cb.jpg)

![A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_Narrator](https://user-images.githubusercontent.com/100189407/186893084-28f4c9f8-44fa-40fc-850a-2ac50fcc2726.jpg)
